pomegranate/_utils.py

```diff
@@ -56,7 +56,10 @@ def _cast_as_tensor(value, dtype=None):
 			return value
 		else:
 			return value.type(dtype)
-
+			
+	if isinstance(value, list) and all(isinstance(v, numpy.ndarray) for v in value):
+		value = numpy.array(value)
```

Comment:
This doesn't seem to have a return statement in all cases?

Reply:
It's saying that if `value` is a list of numpy arrays, cast `value` as a single numpy array. L63 will then take that new numpy array and operate on it, but more efficiently than if it were a list of arrays.

Reply:
Doh!